### PR TITLE
Fix for Issue #123

### PR DESCRIPTION
**Pull request by theperiperi**
changed type() to isinstance() for typechecks in  [src/aijack/defense/paillier/torch_wrapper.py](https://github.com/Koukyosyumei/AIJack/blob/316fcc0c3bbfcd2eaabbf18acf3d27f43b505f70/src/aijack/defense/paillier/torch_wrapper.py)

### CODE REVIEW

**Comment by Koukyosyumei** [comment]
@theperiperi Hi! Thanks for your excellent suggestions! Please check my comments.

**Comment by theperiperi** [comment]
I don't see any comments, where do I check them?

**Review by Koukyosyumei — CHANGES_REQUESTED** [review]

**Comment by Koukyosyumei** [comment]
I'm sorry I forgot to making my comments public. I think now you can see my comments.

File: src/aijack/defense/paillier/torch_wrapper.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ class PaillierTensor(object):
     """torch.Tensor-like object for Paillier Encryption"""
 
     def __init__(self, paillier_array):
-        if type(paillier_array) == list:
+        if isinstance(paillier_array) == list:
```
[the remark this answers is not in the document]
Comment (theperiperi) [reply src/aijack/defense/paillier/torch_wrapper.py]: yes,fixed it

**Review by Koukyosyumei — CHANGES_REQUESTED** [review]
Please check the document of isinstance. https://docs.python.org/3/library/functions.html#isinstance

**Review by Koukyosyumei — APPROVED** [review]

**Pull request merged**